plugins/qtAccessionPlugin/lib/model/QubitAccession.php

```diff
@@ -226,4 +226,22 @@ public function getAlternativeIdentifiers()
 
     return $otherNames;
   }
+
+  /**
+   * Get events
+   */
+  public function getEvents()
+  {
+    $events = [];
```

Comment:
I think you can just call `return $this->_get('accessionEvents')` here - it's implemented as part of the Propel model in the `BaseAccession::_get()` method.

Reply:
Nice!